### PR TITLE
Save generated stamp to file

### PR DESCRIPTION
**Pull request by KomalBhadani27**
When project is run from CLI, save generated_stamp to file.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -41,9 +45,19 @@ def get_user_input():
...
     try:
         generated_stamp = _stampifier.stampify()
-        print(generated_stamp)
     except Error as err:
         LOGGER.debug(err.message)
+
+    if generated_stamp:
+        # Save the generated_stamp to file
+        try:
+            os.makedirs(STAMP_DIRECTORY, exist_ok=True)
```
Comment (detel) [inline main.py:58]: You can merge this try block with the upper one

File: main.py [hunk 2]
```diff
@@ -10,6 +11,9 @@
 LOG_FILENAME = 'website.log'
 logging.basicConfig(filename=LOG_FILENAME, level=logging.DEBUG)
 
+STAMP_DIRECTORY = 'output/'
+STAMP_FILE = 'generated_stamp.html'
```
Comment (detel) [inline main.py:15]: This can be title of the webpage.

Comment (detel) [reply main.py:15]: Will be helpful when stampify takes multiple urls as input.

File: main.py (hunk 1 above)
Comment (KomalBhadani27) [reply main.py:55]: Makes sense! Updated. Let me know if it requires further changes.

File: main.py (hunk 2 above)
Comment (KomalBhadani27) [reply main.py]: Updated accordingly!

File: stampifier.py [hunk 3]
```diff
@@ -56,7 +56,8 @@ def __convert_website_to_stamp(self):
 
         LOGGER.debug(generated_stamp)
 
-        return generated_stamp
+        return generated_stamp, \
```
Comment (detel) [inline stampifier.py:59]: It better to have a data model as `StampifierOutput`

Comment (detel) [reply stampifier.py:59]: `StampifierResult`

Comment (KomalBhadani27) [reply stampifier.py]: Updated! Thanks for suggestion

File: main.py [hunk 4]
```diff
@@ -28,22 +45,24 @@ def get_user_input():
...
 
     try:
-        generated_stamp = _stampifier.stampify()
-        print(generated_stamp)
+        stampifier_output = _stampifier.stampify()
+
+        stamp_file = stampifier_output.stamp_title.replace(' ', '_') \
+            + HTML_EXTENSION
+
+        if stampifier_output.stamp_html:
```
Comment (detel) [inline main.py:59]: You can remove this if condition of that we can throw error and show error UI.

Comment (KomalBhadani27) [reply main.py:59]: My bad! Missed this before merging! Will consider this in future iterations. Thanks for suggestion!

**Pull request merged**